keps/sig-apps/4212-improve-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: KEP Template
```

Comment:
Needs to match the title from https://github.com/kubernetes/enhancements/issues/4212